diesel_derives/src/sql_function.rs

```diff
@@ -302,6 +302,66 @@ pub(crate) fn expand(input: SqlFunctionDecl) -> Result<TokenStream, Diagnostic>
                 }
             };
         }
+
+        if cfg!(feature = "sqlite")
+            && type_args.is_empty()
+            && arg_name.is_empty()
+            && !sql_name.starts_with("pg")
```

Comment:
This line is quite hacky. It's here to skip

```rust
sql_function! { fn pg_my_temp_schema() -> Oid; }
```

which uses a return type only known to Postgres. It would be cleaner to skip functions that contain types unknown to Sqlite. Is there a mechanism to check that? Btw, this issue also exists for functions taking more than zero arguments as soon as they use types unknown to Sqlite.

Reply:
> I would be fine to implement it that way, as long as it's the same for functions with all arities.

I would include such a fix for the newly introduced functions here and fix the existing functions in a separate PR. I like to work with a narrow scope because it's more efficient for the reviewer and for me. Does that sound good for you?

Reply:
@weiznich, fixed.